extension/chrome/background.js

```diff
@@ -1,10 +1,56 @@
+const setIcon = (isConnected) => {
+  const offImage = 'images/OH Chrome Browser Extension Icon - Off.png';
+  const onImage = 'images/OH Chrome Browser Extension Icon - On.png';
+  let icon = {
+    path: {
+      16: offImage,
+      48: offImage,
+      128: offImage,
+    },
+  };
+  if (isConnected) {
+    icon = {
+      path: {
+        16: onImage,
+        48: onImage,
+        128: onImage,
+      },
+    };
+  }
+  chrome.browserAction.setIcon(icon);
+};
+
+const setPopup = (isConnected) => {
+  const connectedHTML = 'popup/connected.html';
+  const disconnectedHTML = 'popup/disconnected.html';
+
+  let popUp = {
+    popup: disconnectedHTML,
+  };
+
+  if (isConnected) {
+    popUp = {
+      popup: connectedHTML,
+    };
+  }
+
+  chrome.browserAction.setPopup(popUp);
+};
+
+const backgroundAction = (isConnected) => {
+  setIcon(isConnected);
+  setPopup(isConnected);
+};
+
 let timeout = 2;
 let connect = () => {
   console.log('connecting');
```

Comment:
I know this isn't your addition but should we be console logging in these extensions?